cypress/integration/testBorna/test.cy.js

```diff
@@ -0,0 +1,179 @@
+        cy.get(".mdc-text-field__input").eq(0).type(block_category_name);
+        cy.get(".mdc-text-field__input").eq(1).type(slug_name);
+        cy.get(".mdc-text-field__input").eq(2).type(block_category_name);
+        cy.contains("Save block category").click();
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.contains("New Block").click();
+        cy.contains(slug_name).click();
+        cy.contains("Save Changes").click();
+
+        cy.contains("New Block").should("exist");
+    });
+    /*
+	it("Test creation of page builder block module using the New Block UI button", () => {
+        cy.visit("/page-builder/page-blocks");
+		getPlaceholder("Search blocks").should("exist");
+		containsClick("New Block");
+		containsClick(slug_name);
+		containsClick("Save Changes");
+		cy.contains("New Block").should("exist");
+    });
+	*/
+
+    it("Test the functionality of editing the block category", () => {
+        cy.visit("/page-builder/block-categories");
+        getPlaceholder("Search block categories").should("exist");
+        cy.contains(block_category_name).click();
+        cy.get(".mdc-text-field__input")
+            .eq(2)
+            .type(block_category_name + "1");
+        cy.contains("Save block category").click();
+        cy.contains("New Block").should("exist");
+        cy.wait(1000);
+    });
+
+    it("Test the functionality of the duplicate block button", () => {
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.contains(block_category_name).click();
+        cy.get(
+            ".rmwc-icon.rmwc-icon--component.material-icons.css-8j2ujd-DuplicateButton.eglpe7i6.mdc-icon-button"
+        )
+            .eq(0)
+            .click();
+
+        cy.contains("New Block").should("exist");
+        cy.wait(1000);
+    });
+
+    it("Test the functionality of the import block button", () => {
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.findByTestId("pb-blocks-list-options-menu").click();
+        cy.contains("Import blocks").click();
+        cy.contains("File URL").type(import_url);
+        cy.contains("Continue").click();
+        cy.contains("All blocks have been imported").should("exist");
+        cy.contains("Continue").click();
+        cy.wait(1000);
+    });
+
+    it("Test the functionality of the export block button", () => {
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.contains(block_category_name).click();
+        cy.get(
+            ".rmwc-icon.rmwc-icon--component.material-icons.css-w71tfn-ExportButton.eglpe7i7.mdc-icon-button"
+        )
+            .eq(0)
+            .click();
+
+        cy.contains("Your export is now ready!").should("exist");
+        cy.wait(1000);
```

Comment:
We'll probably want to address this. Adding a second here/a second there could impact the overall test execution when tests are being run in our CI.

Reply:
I fully agree all the cy.wait(1000)'s should be removed. At the moment the tests were crashing without them and they were an easy fix. Looking into how to fix this I would replace them with cy.intercept()'s.